contents/dataVis.md

```diff
@@ -56,15 +56,18 @@ Plotly, PGFPlotsX, PyPlot?
 - marginalhist, marginalkde and marginalscatter
 - corrplot and cornerplot
 
-## `Makie.jl` {#sec:makie}
+## Makie.jl {#sec:makie}
 
-(WIP) ideas for a better openinig line? 
+> From the japanese word Maki-e, which is a technique to sprinkle lacquer with gold and silver powder.
+> Data is the gold and silver of our age, so let's spread it out beautifully on the screen!
+>
+> _Simon Danisch_
```

Comment:
Maybe add who Simon Danisch is?

Something like "creator of Makie.jl"

Reply:
Full line would be, _Simon Danish, Creator of `Makie.jl`_ ?

Reply:
Yes. Or does he have a better title? Creator and lead developer? Creator and Maintainer?